src/ucp/core/ucp_ep.h

```diff
@@ -172,4 +175,22 @@ ucp_md_map_t ucp_ep_config_get_rma_md_map(const ucp_ep_config_key_t *key,
 ucp_md_map_t ucp_ep_config_get_amo_md_map(const ucp_ep_config_key_t *key,
                                           ucp_lane_index_t lane);
 
+static UCS_F_ALWAYS_INLINE
+size_t ucp_ep_config_get_zcopy_auto_thresh(double iovcnt,
```

Comment:
let's put this in C file, this is not fast path, could also avoid adding __STDC_LIMIT_MACROS to gtest